v3/src/components/graph/utilities/pixi-points.ts

```diff
@@ -411,8 +411,11 @@ export class PixiPoints {
     })
   }
 
-  matchPointsToData(caseData: CaseData[], style: IPixiPointStyle) {
-    const texture = this.getPointTexture(style)
+  matchPointsToData(caseData: CaseData[], style: IPixiPointStyle, animateChange = false) {
+    // If change should be animated, we will modify the point radius with a transition after modifying everything else.
+    const currentRadius = this.points[0] ? this.getMetadata(this.points[0]).style.radius : 6
```

Comment:
This is probably a theoretical issue, but is the first point always the source of truth for the point radius? In theory, a user can hover over it, which affects the radius (but I'm not sure if it affects this property or the scale?).

Maybe extracting that to a separate method like `getCurrentPointRadius` together with a short comment could be a bit safer long-term.